### DISCUSSION TITLE
Timing of 'WRF-Hydro application forced with CONUS404-BA' data availability?

### DISCUSSION THREAD

**Discussion by aputman-usgs** [hytest-org/hytest#529] [Data Storage and Access]
Hi there,

I'm wondering about the timing of internal USGS access to data from the WRF-Hydro application forced with CONUS404-BA. When might those data be available at least internally (osn or on-prem)?

thank you,
Annie

**Comment by sfoks** [comment]
@aputman-usgs the 10yr (WY2010-WY2021) or the 40yr (WY1980-WY2022)?

**Reply by aputman-usgs** [reply]
The 10 year for now but am also interested in the 40 year. ty!

**Reply by aubreyd** [reply]
@aputman-usgs: This is the ScienceBase page for the 10-year model archive that is now public, but not sure if this is the format you need...?
https://www.sciencebase.gov/catalog/item/661039a6d34e6334665050f4

**Reply by aputman-usgs** [reply]
Hi @aubreyd - yes, this access point works for us and I've passed it along to our colleagues. This (general) inquiry was the same one for which we (Martha Scholl and I) asked you (offline) about the variables supporting T vs E partitioning. Thank you!
